src/hooks/useAutomateConnection.ts

```diff
@@ -28,8 +28,8 @@ export async function isConnectedToAutomate(ethereum: any): Promise<boolean> {
         },
       ],
     });
-    return res.client === 'automate';
```

Comment:
While you may want to rely on `res.params.network` to fail if this is not an automate connection, it's just too indirect. It's better to be more explicit and add this extra check.